mllib/src/main/scala/org/apache/spark/mllib/feature/Word2Vec.scala

```diff
@@ -555,6 +614,14 @@ class Word2VecModel private[spark] (
       (word, wordVectors.slice(vectorSize * ind, vectorSize * ind + vectorSize))
     }
   }
+
+  /**
+   * get word vector array. length is vocabularySize*vectorSize
+   * @return the array of word vectors, need to split it by vectorSize to get each individual vector
+   */
+  def getWordVectors: Array[Float] = {
```

Comment:
Yes, let's remove other changes that are not related to this issue. I don't think this class should expose its internal data structures and utility functions "just because". Just the sentence length change should be here